Gemfile

```diff
@@ -38,7 +39,10 @@ end
 
 group :test do
   gem "capybara", ">= 2.15"
+  gem "database_cleaner-active_record"
   gem "selenium-webdriver"
+  gem "shoulda-matchers"
+  gem "simplecov", require: false
```

Comment:
hỗ trợ tính toán cho bạn những dòng code nào đã được cover test, những dòng nào chưa